.github/workflows/swift-qltest.yml

```diff
@@ -32,6 +32,9 @@ jobs:
       - name: Build Swift extractor
         run: |
           bazel run //swift:create-extractor-pack
+        env:
+          CC: clang
+          CXX: clang++
```

Comment:
instead of this you should write in `.bazelrc`
```
build --repo_env=CC=clang  --repo_env=CXX=clang++
```
so that it also auto applies on dev builds

Reply:
and I'll link [this comment](https://github.com/github/codeql/pull/8788/files#r858429814) here as the PR UI might hide it

Reply:
@redsun82 wouldn't it break cases on Ubuntu with versioned binary names, e.g. `clang-12`?

Reply:
you can always override that creating a `local.bazelrc`, and ubuntu will have a link to a default clang version